OxfordBodleian/Juel-Jensen/BDLaethb5.xml

```diff
@@ -281,21 +9333,342 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             <textClass>
                 <keywords scheme="#ethioauthlist">
                     <term key="ChristianLiterature"/>
+                    <term key="Hagiography"/>
                 </keywords>
             </textClass>
             <langUsage>
                 <language ident="en">English</language>
                 <language ident="gez">Gǝʿǝz</language>
+                <language ident="la">Latin</language>
             </langUsage>
         </profileDesc>
         <revisionDesc>
             <change who="DR" when="2020-01-20">Created record</change>
             <change who="DR" when="2020-02-24">Added dimensions and layout</change>
-        <change who="SH" when="2020-04-22">Added editors</change></revisionDesc>
+            <change who="SH" when="2020-04-22">Added editors</change>
+            <change who="DR" when="2020-09-10">Continued description</change>
+            <change who="DR" when="2020-09-15">Added corrections provided by Massimo Villa</change>
+        </revisionDesc>
     </teiHeader>
     <text xml:base="https://betamasaheft.eu/">
         <body>
-            <div type="bibliography"/>
+            <div type="edition">
```

Comment:
Under this section you collected all text parts that are provided in their entirety. I would need help in coorecting my old files to this new system.